DQMOffline/L1Trigger/src/L1TTauOffline.cc

```diff
@@ -678,13 +690,15 @@ void L1TTauOffline::getProbeTaus(const edm::Event& iEvent,
         }
       }
       {
-        const edm::Provenance* prov = antiele.provenance();
-        const std::vector<std::string> psetsFromProvenance =
-            edm::parameterSet(prov->stable(), iEvent.processHistory())
-                .getParameter<std::vector<std::string>>("workingPoints");
-        for (uint i = 0; i < psetsFromProvenance.size(); i++) {
-          if (psetsFromProvenance[i] == AntiEleWP_)
-            AntiEleWPIndex_ = i;
+        if (m_AntiEleExists) {
+          const edm::Provenance* prov = antiele.provenance();
+          const std::vector<std::string> psetsFromProvenance =
+              edm::parameterSet(prov->stable(), iEvent.processHistory())
+                  .getParameter<std::vector<std::string>>("workingPoints");
+          for (uint i = 0; i < psetsFromProvenance.size(); i++) {
+            if (psetsFromProvenance[i] == AntiEleWP_)
+              AntiEleWPIndex_ = i;
```

Comment:
Here you are comparing an `int` (`AntiEleWPIndex_`) with a `uint`: please make uniform
I would stick on `unsigned int` (preferred to `uint`), or even `size_type` for all them
To be propagated also to `AntiMuWPIndex_`, quite likely